p4-16/spec/P4-16-spec.mdk

```diff
@@ -1828,6 +1828,7 @@ P4 supports the following built-in base types:
 baseType
     : BOOL
     | ERROR
+    | MATCH_KIND
```

Comment:
Minor nit-pick, but should we go ahead and make the relative order of the other alternatives `BIT`, `INT`, and `STRING` be in the same order as each other between the grammar.mdk and spec files, too?

I see that `match_kind` was already mentioned in the text of the spec in Section 7.1, so this does look like it was simply an omission in the grammar.